src/api/app/views/webui2/webui/search/index.html.haml

```diff
@@ -1,6 +1,47 @@
 - @pagetitle = 'Search'
+= content_for :ready_function do
+  :plain
+    $('#search-text').focus();
```

Comment:
This could be replaced by `autofocus: true`, so the `text_field_tag` line would be:

= text_field_tag('search_text', params[:search_text], placeholder: 'Search', **autofocus: true,** class: 'form-control')

I would also add `required: true` to the `text_field_tag`, since not writing anything here is not an option.

Reply:
Ok to the autofocus. But if I'm not wrong, when we want to search by Issue, we can leave the text field empty. So there is no reason to add a 'required: true'. Here you have [an example](https://build.opensuse.org/search?utf8=%E2%9C%93&search_text=&commit=Submit+Query&issue_tracker=bnc&issue=1051510&project=1&package=1&name=1&attrib_type_id=).

Reply:
Oh true, for the search by issue.